less/home-page.less

```diff
@@ -1 +1,89 @@
-// Home Content
\ No newline at end of file
+// Home Content
+
+.container-content{
+    max-width: 880px;
+    margin: 0px auto;
+
+    .headerimg{
+        width:100%;
+        height: 200px;
+    }
+
+    .content-intro, .content-main, .content-signup {
+        margin: 20px;
+        h2{
+            font-size:2rem;
+            padding: 10px 0px;
+        }
+
+        p{
+            padding: 10px 0px;
+            line-height: 150%  
+        }
+    }
+
+    .image-row{
+        margin: 20px;
+        .flexmix(space-between; center;);
+
+        #img1{
+            @media @phone{
+                width: 90%;
+            }
+        }
+        #img2{
```

Comment:
There is typically a space between a selector and the bracket in CSS. Look into extensions such as `Beautify` or `Prettier` they'll clean it up for you.